pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -3598,3 +3598,34 @@ CREATE UNIQUE INDEX ON public.t_118626(c) WHERE n>= 1;
 
 statement ok
 ALTER TABLE public.t_118626 ALTER PRIMARY KEY USING COLUMNS(b);
+
+# Regression test to make sure that we can drop and re-add a CHECK constraint
```

Comment:
Looks like the legacy schema changer doesn't like this test - are we able to specify "do this only in dsc land" in logic tests?

Reply:
there is! i added a skip directive here for the legacy schema changer